server/app.py

```diff
@@ -1535,19 +1544,19 @@ def log_rdo(type=None, contact=None, account=None, customer=None, subscription=N
     sub_meta = subscription["metadata"]
     sub_plan = subscription["plan"]
     customer_id = subscription["customer"]
-    trial_end = subscription["trial_end"]
+    start_date = subscription["start_date"]
     donation_type_info = DONATION_TYPE_INFO[type]
     installment_period = "yearly" if sub_plan["interval"] == "year" else "monthly"
 
     if type == "business_membership" and account:
-        rdo = RDO(account=account)
+        rdo = RDO(account=account, date=start_date)
         year = datetime.now(tz=ZONE).strftime("%Y")
         rdo.name = f"{year} Business {account.name} Recurring"
         rdo.record_type_name = "Business Membership"
         rdo.installments = None
 
     else:
-        rdo = RDO(contact=contact, date=trial_end)
+        rdo = RDO(contact=contact, date=start_date)
```

Comment:
**question:** Just more curious than anything, but what does this do?

Reply:
This code is passing the start_date to the RDO so that the RDO can better reflect the subscription it's dealing with. This is important in cases where we're syncing an old subscription that never made it to Salesforce so that the recurring dates in SF line up with the recurring dates in Stripe. Currently, the RDO just takes today's date (or the trial_end as you see there, but that's no longer relevant).